fintech-examples/fintech-server/pom.xml

```diff
@@ -59,6 +59,31 @@
             <version>5.1.7.RELEASE</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>2.25.1</version>
```

Comment:
Avoid hardcoded version numbers, many of these are already declared in parent pom

Reply:
done